gsa/src/web/pages/results/detailspage.js

```diff
@@ -77,6 +76,7 @@ import OverrideComponent from '../overrides/component';
 import TicketComponent from '../tickets/component';
 
 import ResultDetails from './details';
+import NewTicketIcon from 'web/components/icon/newticketicon';
```

Comment:
Imho this shouldn't work. The new Icon component from the previous commit is a TicketIcon but not NewTicketIcon

Reply:
The svg is included in the changes of this pr.

Reply:
Ok, I just found the files. I guess git didn't list the commit properly or I just oversaw.